libs/wcm/context/connectionProvider.test.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useContext } from 'react';
+import { mount } from 'enzyme';
+import ConnectionProvider from './connectionProvider';
+import ConnectionContext from './connectionContext';
+import { EVENTS } from '../constants/lifeCycle';
+
+describe('connectionProvider', () => {
+  it('Should mount a child with access to the default context', () => {
+    const TestComponent = () => {
+      const { session } = useContext(ConnectionContext);
+      return (
+        <div>
+          {Object.keys(session).map(key => (<span key={key}>{key}</span>))}
+        </div>
+      );
+    };
+    const wrapper = mount(
+      <ConnectionProvider>
+        <TestComponent />
+      </ConnectionProvider>,
+    );
+    expect(wrapper.find('span').at(0).text()).toBe('request');
+    expect(wrapper.find('span').at(1).text()).toBe('data');
+    expect(wrapper.find('span').at(2).text()).toBe('loaded');
```

Comment:
I think this can be 

```suggestion
     Object.keys(session).forEach((key, index)=> {
       expect(wrapper.find('span').at(index).text()).toBe(key)
     });
```

I think since the idea is to test that the `span` displays the session values from context. Also it means if these keys change in the future, there would be no required need to change it here